includes/Dashboard/Templates/Settings.php

```diff
@@ -193,71 +193,60 @@ public function load_store_content() {
-         * meta key 'dokan_profile_settings' for that Payment method
-         *
-         * @since 3.4.3
-         */
-        $method_key_to_storage_key = apply_filters( 'dokan_payment_method_storage_key', [] );
-
-        $used_methods = $profile_info['payment'];
-
-        foreach ( $methods as $method_key => $method ) {
-            if ( empty( $method ) ) {
-                unset( $methods[ $method_key ] );
+        $payment_method_ids = array_keys( $payment_methods );
 
-                if ( isset( $used_methods[ $method_key ] ) ) {
-                    unset( $used_methods[ $method_key ] );
-                } else if ( isset( $method_key_to_storage_key[ $method_key ] ) && isset( $used_methods[ $method_key_to_storage_key[ $method_key ] ] ) ) {
-                    unset( $used_methods[ $method_key_to_storage_key[ $method_key ] ] );
-                }
+        $seller_id = dokan_get_current_user_id();
 
-                if ( isset( $method_key_to_storage_key[ $method_key ] ) ) {
-                    unset( $method_key_to_storage_key[ $method_key ] );
-                }
+        $seller_connected_payment_method_ids = array_filter(
+            $payment_method_ids,
+            function ( $payment_method_id ) use( $seller_id ) {
+                return $this->is_seller_connected( $payment_method_id, $seller_id );
             }
-        }
+        );
 
-        list( $used_method_keys, $unused_method_keys ) = $this->get_separated_method_keys( $methods, $used_methods, $method_key_to_storage_key );
+        $seller_disconnected_payment_method_ids = array_diff( $payment_method_ids, $seller_connected_payment_method_ids );
 
-        $unused_methods = $this->get_payment_methods( $unused_method_keys );
-        $used_methods   = $this->get_payment_methods( $used_method_keys );
+        $seller_disconnected_payment_methods = $this->get_payment_methods( $seller_disconnected_payment_method_ids );
+        $seller_connected_payment_methods    = $this->get_payment_methods( $seller_connected_payment_method_ids );
 
         $method_key = str_replace( '/manage-', '', $slug_suffix ); // if we are requesting a single payment method page(to edit or for first time setup) then we have the corresponding payment method key in the url
 
-        if ( stripos( $method_key, '/edit' ) !== false ) { // if payment method key has /edit suffix then we are trying to edit the method, otherwise we are doing a initial setup for that payment method
+        $is_edit_mode = false;
+
+        if ( false !== stripos( $method_key, '/edit' ) ) { // if payment method key has /edit suffix then we are trying to edit the method, otherwise we are doing a initial setup for that payment method
             $is_edit_mode = true;
             $method_key   = str_replace( '/edit', '', $method_key ); // removing /edit suffix to get payment method key
         }
 
+        $profile_info = get_user_meta( get_current_user_id(), 'dokan_profile_settings', true );
```

Comment:
Why you're using `get_current_user_id()` here? Why not `$seller_id = dokan_get_current_user_id();`?